From 5248adb1fbbb50de64d76825fae4e7a02e0d5e5d Mon Sep 17 00:00:00 2001
From: William Woodall <wjwwood@gmail.com>
Date: Tue, 12 Mar 2013 16:15:37 -0700
Subject: [PATCH] Adding install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d636ba2..47e4512 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,5 +57,8 @@ if (NOT MSVC)
   install(FILES ${pkg_conf_file} DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
 endif()
 
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/urdfdom)
+
 
 message(STATUS "Configuration successful. Type make to compile urdfdom")
-- 
1.7.12.4 (Apple Git-37)

