From bccce42de88ed7dbc3597e90b46ed4997b61c35f Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 19:53:54 -0800
Subject: [PATCH] add install rule for package.xml

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2ed60b8..48b756e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,10 +71,13 @@ if (NOT MSVC)
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig/ COMPONENT pkgconfig)
 endif()
 
-add_subdirectory(urdf_parser_py)
+#add_subdirectory(urdf_parser_py)
 
 message(STATUS "Configuration successful. Type make to compile urdfdom")
 
 SET_DIRECTORY_PROPERTIES(PROPERTIES
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/cmake/urdfdom-config.cmake
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/cmake/pkgconfig/urdfdom.pc)
+
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/urdfdom)
-- 
1.9.0.rc1.175.g0b1dcb5

