From 00584ffe0e87d9339f18239de3e09c5ea7824cac Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@willowgarage.com>
Date: Tue, 18 Jun 2013 23:30:19 +0300
Subject: [PATCH 1/2] install rule for package.xml

---
 CMakeLists.txt |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7878323..c0c5fe8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,5 +57,7 @@ if (NOT MSVC)
   install(FILES ${pkg_conf_file} DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
 endif()
 
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/urdfdom)
 
 message(STATUS "Configuration successful. Type make to compile urdfdom")
-- 
1.7.9.5

