From 808eb42cf79bcb8af363bc228650ccda3d347d5d Mon Sep 17 00:00:00 2001
From: isucan <isucan@gmail.com>
Date: Thu, 5 Sep 2013 01:31:29 +0300
Subject: [PATCH] Added install rule for package.xml

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0386233..33b36b0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,10 +68,13 @@ if (NOT MSVC)
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig/ COMPONENT pkgconfig)
 endif()
 
-add_subdirectory(urdf_parser_py)
+#add_subdirectory(urdf_parser_py)
 
 message(STATUS "Configuration successful. Type make to compile urdfdom")
 
 SET_DIRECTORY_PROPERTIES(PROPERTIES
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/cmake/urdfdom-config.cmake
   ADDITIONAL_MAKE_CLEAN_FILES ${CMAKE_BINARY_DIR}/cmake/pkgconfig/urdfdom.pc)
+
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/urdfdom)
-- 
1.8.5.1

