From deea8c3394c1e295d13d7844d353c61bc366f624 Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@willowgarage.com>
Date: Wed, 19 Jun 2013 01:26:00 +0300
Subject: [PATCH 2/2] install as catkin package

---
 CMakeLists.txt |   32 +++++++++++++++++++++++++-------
 1 file changed, 25 insertions(+), 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c0c5fe8..09dd942 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,14 +38,34 @@ find_package(Boost REQUIRED thread)
 include_directories(${Boost_INCLUDE_DIR})
 link_directories(${Boost_LIBRARY_DIRS})
 
+set(${PROJECT_NAME}_inc_dirs urdf_parser/include)
+find_package(catkin REQUIRED)
+catkin_package(
+  INCLUDE_DIRS ${${PROJECT_NAME}_inc_dirs}
+  LIBRARIES urdfdom_model urdfdom_model_state urdfdom_sensor urdfdom_world
+  DEPENDS console_bridge urdfdom_headers
+  DEPENDS Boost COMPONENTS thread
+)
+include_directories(${${PROJECT_NAME}_inc_dirs})
+
+# split version in parts and pass to preprocessor
+string(REPLACE "." ";" ${PROJECT_NAME}_VERSION_LIST "${${PROJECT_NAME}_VERSION}")
+list(LENGTH ${PROJECT_NAME}_VERSION_LIST _count)
+if(NOT _count EQUAL 3)
+  message(FATAL_ERROR "${PROJECT_NAME} version '${${PROJECT_NAME}_VERSION}' does not match 'MAJOR.MINOR.PATCH' pattern")
+endif()
+list(GET ${PROJECT_NAME}_VERSION_LIST 0 URDF_MAJOR_VERSION)
+list(GET ${PROJECT_NAME}_VERSION_LIST 1 URDF_MINOR_VERSION)
+list(GET ${PROJECT_NAME}_VERSION_LIST 2 URDF_PATCH_VERSION)
+
 add_subdirectory(urdf_parser)
 
 set(PKG_NAME ${PROJECT_NAME})
 set(PKG_LIBRARIES urdfdom_sensor urdfdom_model_state urdfdom_model urdfdom_world)
 set(PKG_DEPENDS urdfdom_headers console_bridge)
 set(cmake_conf_file "${CMAKE_CURRENT_SOURCE_DIR}/cmake/urdfdom-config.cmake")
-configure_file("${cmake_conf_file}.in" "${cmake_conf_file}" @ONLY)
-install(FILES ${cmake_conf_file} DESTINATION share/${PROJECT_NAME}/cmake/ COMPONENT cmake)
+#configure_file("${cmake_conf_file}.in" "${cmake_conf_file}" @ONLY)
+#install(FILES ${cmake_conf_file} DESTINATION share/${PROJECT_NAME}/cmake/ COMPONENT cmake)
 
 # Make the package config file
 if (NOT MSVC)
@@ -53,11 +73,9 @@ if (NOT MSVC)
   set(PKG_DEPENDS "urdfdom_headers console_bridge") # make the list separated by spaces instead of ;
   set(PKG_URDF_LIBS "-lurdfdom_sensor -lurdfdom_model_state -lurdfdom_model -lurdfdom_world")
   set(pkg_conf_file "${CMAKE_CURRENT_SOURCE_DIR}/cmake/pkgconfig/urdfdom.pc")
-  configure_file("${pkg_conf_file}.in" "${pkg_conf_file}" @ONLY)
-  install(FILES ${pkg_conf_file} DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
+  #configure_file("${pkg_conf_file}.in" "${pkg_conf_file}" @ONLY)
+  #install(FILES ${pkg_conf_file} DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
 endif()
 
-# Install catkin package.xml
-install(FILES package.xml DESTINATION share/urdfdom)
 
-message(STATUS "Configuration successful. Type make to compile urdfdom")
+#message(STATUS "Configuration successful. Type make to compile urdfdom")
-- 
1.7.9.5

